src/atoms/BylineImage.js

```diff
@@ -1,6 +1,14 @@
 import styled from 'styled-components';
 
 const BylineImage = styled.img`
+	width: 6.4rem;
+   height: 6.4rem;
```

Comment:
Is something weird about this indentation?

Reply:
Yes, there was something weird. It looked good in my editor but it is fixed now.